### PR TITLE
Test Lab Python sample

### PR DESCRIPTION
**Pull request by kevinthecheung**

### CODE REVIEW

**Comment by kevinthecheung** [comment]
@jhuleatt I have not tested this.

File: Python/testlab-to-slack/main.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+# The requests library to send web requests to Slack.
+import requests
+# [END import]
+
+
+# [START postToSlack]
+def post_to_slack(title: str, details: str) -> any:
+    """Posts a message to Slack via a Webhook."""
+    response = requests.post(
+        url=os.environ["SLACK_WEBHOOK_URL"],
```
Comment (jhuleatt) [inline Python/testlab-to-slack/main.py:31]: Can you please use params for this? https://github.com/firebase/firebase-functions-python/pull/2/files

We should eventually have it as a Secret, but secret params aren't supported in Python yet

Comment (jhuleatt) [reply Python/testlab-to-slack/main.py:31]: hm, actually the Node sample uses `process.env` at the moment as well. Changing to params will probably change the narrative in the docs, so let's stick to env vars as you have it now

**Review by jhuleatt — APPROVED** [review]
Thank you @kevinthecheung

**Review by jhuleatt — CHANGES_REQUESTED** [review]
Let's make sure we get a chance to test this before merging

**Comment by kevinthecheung** [comment]
> Let's make sure we get a chance to test this before merging

Tested, and it works.

**Pull request merged**